src/braket/ahs/driving_field.py

```diff
@@ -95,11 +95,11 @@ def stitch(
             other (DrivingField): The second shifting field to be stitched with.
             boundary (StitchBoundaryCondition): {"mean", "left", "right"}. Boundary point handler.
                 Possible options are
-            * "mean" - take the average of the boundary value points of the first
-              and the second time series.
-            * "left" - use the last value from the left time series as the boundary point.
-            * "right" - use the first value from the right time series as the boundary
-              point.
+                * "mean" - take the average of the boundary value points of the first
```

Comment:
Did you check the output of this? If not, can you convert the '*' to '-' and indent by 2?

Reply:
Missed out checking this. Fixed it by converting to `-`, Thanks!